tasks/main.yml

```diff
@@ -29,7 +29,10 @@
   template:
     src: tinc.systemd.service.j2
     dest: /etc/systemd/system/tinc.service
-  notify: Restart Service
+  notify:
```

Comment:
see my comment on the simplification above, and the fact to add it in a different commit. I like where you're heading to, though :)